core/logger/logger.go

```diff
@@ -115,6 +120,20 @@ type Logger interface {
 	Recover(panicErr interface{})
 }
 
+type Config struct {
+	LogLevel       zapcore.Level
+	Dir            string
+	JsonConsole    bool
+	UnixTS         bool
+	FileMaxSizeMB  int
+	FileMaxAgeDays int
+	FileMaxBackups int // files
+	Hostname       string
+	ChainlinkDev   bool
+	SplunkToken    string // enables splunk logging if not empty
+	SplunkURL      string
+}
```

Comment:
nit/ This was co-located with its constructor and other methods before (`(c *Config) New()`) - could we move it back down there?

Reply:
Yup moved back now